apps/web/src/pages/Communities/CommunityMembersPage/CommunityMembersPage.tsx

```diff
@@ -125,9 +133,9 @@ const CommunityMembers = ({ communityQuery }: CommunityMembersProps) => {
         search={{
           internal: true,
           label: intl.formatMessage({
-            defaultMessage: "Search community members",
-            id: "33oume",
-            description: "Label for the community members table search input",
+            defaultMessage: "Search by keyword",
+            id: "PYMFoh",
+            description: "Label for the keyword search input",
```

Comment:
Not sure if this is the correct place for it but, we have a common message for this under `~/messages/adminMessages`. If we are consolidating this message on "by keyword" for all tables maybe we make it the default label?

https://github.com/GCTC-NTGC/gc-digital-talent/blob/1686a9caa9cd5a42d05a04dfe43cb51b10aa62cb/apps/web/src/messages/adminMessages.ts#L212-L216

Also not the place for this discussion but I feel like we are losing some context with this change :sweat_smile: . From a screen reader perspective, it is no longer explicit what you are searching. When testing this, I found it less helpful to know I can "search by keyword" (which is not entirely accurate) than to know what dataset I am searching (i,.e communities).

Reply:
Yeah, valid concerns for sure.  I wouldn't be surprised if we went back or started using hidden text for more context.  In the meantime, I consolidate the use to the adminMessages file.  Since I'm already touching up departments and skills I updated it there, too.
[consolidate search by keyword](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12005/commits/32ef4f407e2d33283b3569487fc41d60048845b6)